diff --git a/node_modules/jest-runner/build/runTest.js b/node_modules/jest-runner/build/runTest.js
index 2c58b4c..fa43919 100644
--- a/node_modules/jest-runner/build/runTest.js
+++ b/node_modules/jest-runner/build/runTest.js
@@ -179,7 +179,11 @@ function freezeConsole(testConsole, config) {
       true
     );
     process.stderr.write(`\n${formattedError}\n`);
-    process.exitCode = 1;
+
+    // Willow patch: We don't want to confusingly fail tests runs where all the
+    // tests passed just because something did an unexpected console log.
+    // Reference https://github.com/facebook/jest/issues/9324
+    // process.exitCode = 1;
   };
 } // Keeping the core of "runTest" as a separate function (as "runTestInternal")
 // is key to be able to detect memory leaks. Since all variables are local to
